gfx_imgui.h

```diff
@@ -90,8 +92,15 @@ class GfxImGuiInternal
 
         uint8_t *font_data;
         int32_t font_width, font_height;
-        if(font_filename != nullptr)
-            io.Fonts->AddFontFromFileTTF(font_filename, 16.0f);
+        for (uint32_t i = 0; i < font_count; ++i)
+        {
+            if (i == 0 && font_configs != nullptr && font_configs[0].MergeMode == true)
+				io.Fonts->AddFontDefault();
```

Comment:
If the first user font has configuration set to merge then we load in the default font and then merge the new fonts glyphs into the same glyph atlas. This allows adding additional glyphs to the default font as that only supports basic Latin characters.
If not set to merge then the new font will become the new default